packages/asset-swapper/test/dex_sampler_test.ts

```diff
@@ -336,6 +369,11 @@ describe('DexSampler tests', () => {
                     expect(fillAmounts).to.deep.eq(expectedTakerFillAmounts);
                     return fillAmounts.map(a => a.times(ratesBySource[ERC20BridgeSource.Eth2Dai]).integerValue());
                 },
+                sampleSellsFromUniswapV2: (path, fillAmounts) => {
+                    expect(path).to.deep.eq([expectedMakerToken, expectedTakerToken]);
```

Comment:
Can I get a sanity check that the path **is* maker -> taker for sell quotes, and taker -> maker for buy quotes?